### PR TITLE
Fix G2 order and format

### PR DESCRIPTION
**Pull request by karst**
PR #19446 did not follow the correct G2 order so this is fixing that.

Also adds the added icons to the [Plugin] chart as should when adding icons to G2.

### CODE REVIEW

**Comment by duncanspumpkin** [comment]
What makes this order anymore correct than the old one. This is the first time adding palette maps.

**Comment by spacek531** [comment]
More correct or not, it would be nice if track sprites were at the end of g2, since X7123M3-256's track rendering tool appends sprites to the end. Moving the color sprites before track sprites makes it easier to make track types.

**Comment by karst** [comment]
The icons were not placed with the icons, and indeed I'd like to have the tracks at the end. It overall keeps it cleaner.

**Review by duncanspumpkin — APPROVED** [review]

File: distribution/openrct2.d.ts [hunk 1]
```diff
@@ -3869,7 +3869,7 @@ declare global {
         "map_north_pressed" | "map_south" | "map_south_pressed" | "map_west" | "map_west_pressed" |
         "mountain_tool_even" | "mountain_tool_odd" | "multiplayer" | "multiplayer_desync" | "multiplayer_sync" |
         "multiplayer_toolbar" | "multiplayer_toolbar_pressed" | "mute" | "mute_pressed" | "news_messages" |
-        "normal_selection_6x6" | "paste" | "path_railings" | "path_surfaces" | "paths" | "placeholder" |
+        "normal_selection_6x6" | "palette_invisible" | "palette_invisible_pressed" | "paste" | "path_railings" | "path_surfaces" | "paths" | "placeholder" |
```
Comment (duncanspumpkin) [inline distribution/openrct2.d.ts:3872]: Its not really the palette here is just an icon to represent the palette. Are we confident that people will understand this?

Comment (karst) [reply distribution/openrct2.d.ts:3872]: perhaps I could change it to `palette_select_invisible`, `palette_select_invisible_pressed`?

Comment (karst) [reply distribution/openrct2.d.ts]: applied the previously mentioned change.

**Comment by karst** [comment]
I am extremely confused. This hasn't been merged yet, but all changes in develop it seems?

**Comment by karst** [comment]
This was merged with #20226 it seems. I don't know how that happened.

**Pull request closed without merging**